README.md

```diff
@@ -2149,6 +2149,9 @@ If you run `terragrunt hclfmt` at the `root`, this will update:
 - `root/dev/terragrunt.hcl`
 - `root/qa/terragrunt.hcl`
 
+Additionally, there's a flag `--terragrunt-check`. It allows to validating if files are properly formatted. It does not
```

Comment:
```suggestion
Additionally, there's a flag `--terragrunt-check`. It can be used for validating if files are properly formatted without actually formatting the files. It does not
```